pom.xml

```diff
@@ -301,6 +301,8 @@
         <javacpp.version>1.4.4</javacpp.version>
         <javacpp-presets.version>1.4.4</javacpp-presets.version>
         <javacv.version>1.4.4</javacv.version>
+
+        <cpython-platform.version>3.6-1.4.4-SNAPSHOT</cpython-platform.version>
```

Comment:
1.4.4 has been released, so we can remove the SNAPSHOT bit! And actually just use `${javacpp-presets.version}` to have it take the same version as all the other presets.